lib/alerts/onboarding_alert_manager.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class OnboardingAlertManager
+  def create_alert(user_name, msg)
+    user = User.find_by(username: user_name)
+    unless Alert.exists?(user_id: user.id, type: 'OnboardingAlert')
```

Comment:
I think this check is not helpful. It'll be better to create a second alert if the same user goes through onboarding again and submits a new response.